### DISCUSSION TITLE
part-ii/2021-69/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#356] [Question comments]
Part II 2021 Galois Theory: Paper 2, Section II, 18I 

https://questions.tripos.org/part-ii/2021-69/

**Comment by artie2000** [comment]
(i) is bookwork.

<details>
<summary>(ii)</summary>

The splitting field is $L'=\mathbb{F}_q^n$, where $n=\text{lcm}(\deg h_1,\dots,\deg h_r)$.

By uniqueness of finite fields, $f$ splits in $L'$. Then the splitting field of $f$ has degree a multiple of each $\deg h_i$ by the tower law, so $[L:\mathbb{F}_q]\mid[L':\mathbb{F}_q]$. Apply uniqueness again to get the inclusion.

</details>

<details>
<summary>(iii)</summary>

Finite fields are cyclic. Have $m=q^n-1$. Since the Galois group is generated by $x\to x^q$, the image is

 $\{q^k\mid 0\leq k<n\}$.

</details>

<details>
<summary>(b)</summary>

The resolvent cubic is $x^3-48x-64$, which is irreducible.

Therefore $\text{Gal}(f)=A_4$.

</details>

**Comment by Kiolt** [comment]
<details>
<summary>(b)</summary>

Dedekind with $p=7$ produces a 3-cycle, so $A_4$.

</details>

**Reply by artie2000** [reply]
This way is much easier
